### ISSUE TITLE
e is null

### ISSUE THREAD

**Issue by pirona** [Graylog2/graylog2-server#6516]
## Expected Behavior
Being able to consult any streams, as usual.

## Current Behavior
This errors happens at login for me, but this is pretty much the only thing I can write here. Didn't see anything relevant in server.log

Any stream page displays the monkey 404 error and tells me the following (obviously I replaced my own urls): 

```
e is null

Stack Trace:

_updateHeight@https://graylog.example.com/assets/LoggedInPage.60a2621478c6c8ca385c.js:1:187203
componentDidMount@https://graylog.example.com/assets/LoggedInPage.60a2621478c6c8ca385c.js:1:186837
_/<@https://graylog.example.com/assets/vendor.91c91d4a31d54d96392a.js:73:88989
Ro@https://graylog.example.com/assets/vendor.91c91d4a31d54d96392a.js:18:82395
Xo@https://graylog.example.com/assets/vendor.91c91d4a31d54d96392a.js:18:85070
Os/<@https://graylog.example.com/assets/vendor.91c91d4a31d54d96392a.js:18:98277
29/</exports.unstable_runWithPriority@https://graylog.example.com/assets/vendor.91c91d4a31d54d96392a.js:35:3284
Os@https://graylog.example.com/assets/vendor.91c91d4a31d54d96392a.js:18:98212
Ys@https://graylog.example.com/assets/vendor.91c91d4a31d54d96392a.js:18:97988
Ss@https://graylog.example.com/assets/vendor.91c91d4a31d54d96392a.js:18:97333
Ls@https://graylog.example.com/assets/vendor.91c91d4a31d54d96392a.js:18:96354
Zo@https://graylog.example.com/assets/vendor.91c91d4a31d54d96392a.js:18:95228
enqueueSetState@https://graylog.example.com/assets/vendor.91c91d4a31d54d96392a.js:18:44755
24/b.prototype.setState@https://graylog.example.com/assets/vendor.91c91d4a31d54d96392a.js:26:1665
_refreshData/this.promise</<@https://graylog.example.com/assets/611727de-75.60a2621478c6c8ca385c.js:1:2875
l@https://graylog.example.com/assets/builtins.60a2621478c6c8ca385c.js:104:88608
YPyIcTeH/</</<[22]</n.exports/O.prototype._settlePromiseFromHandler@https://graylog.example.com/assets/builtins.60a2621478c6c8ca385c.js:104:61890
YPyIcTeH/</</<[22]</n.exports/O.prototype._settlePromise@https://graylog.example.com/assets/builtins.60a2621478c6c8ca385c.js:104:62690
YPyIcTeH/</</<[22]</n.exports/O.prototype._settlePromise0@https://graylog.example.com/assets/builtins.60a2621478c6c8ca385c.js:104:63389
YPyIcTeH/</</<[22]</n.exports/O.prototype._settlePromises@https://graylog.example.com/assets/builtins.60a2621478c6c8ca385c.js:104:64716
YPyIcTeH/</</<[2]</s.prototype._drainQueue@https://graylog.example.com/assets/builtins.60a2621478c6c8ca385c.js:104:18582
YPyIcTeH/</</<[2]</s.prototype._drainQueues@https://graylog.example.com/assets/builtins.60a2621478c6c8ca385c.js:104:18643
s/this.drainQueues@https://graylog.example.com/assets/builtins.60a2621478c6c8ca385c.js:104:16866
u@https://graylog.example.com/assets/polyfill.60a2621478c6c8ca385c.js:1:33700
M/<@https://graylog.example.com/assets/polyfill.60a2621478c6c8ca385c.js:1:33845
f@https://graylog.example.com/assets/polyfill.60a2621478c6c8ca385c.js:1:20081


Component Stack:

    in DecoratorSidebar
    in div
    in t
    in t
    in div
    in t
    in div
    in t
    in t
    in Uncontrolled(t)
    in div
    in t
    in div
    in AutoAffix
    in SearchSidebar
    in div
    in t
    in div
    in t
    in t
    in t
    in SearchPage
    in Unknown
    in n
    in div
    in t
    in div
    in t
    in div
    in AppWithSearchBar
    in div
    in t
    in t
    in withRouter(t)
    in div
    in App
    in RouterContext
    in Router
    in h
    in t
    in n
    in AppFacade
```

## Possible Solution
I had the problem twice with two different Graylog versions : 
I'm not really not sure about those solutions : 

version 3.1 : service restart did the trick : I had access to streams afterward
version 3.1.2 : I restarted the service, then stopped-started it, then rotated indices... with a bit of waiting in between, so I'm  not sure of what worked, sorry.

## Steps to Reproduce (for bugs)
Not sure either here ..
Just connected using when I arrived at work with any user with privileges.

## Context
Nothing special really, just browse at logs.

## Your Environment
One graylog node, and using an extra Haproxy to manage TLS (certificate and key).

* Graylog Version: 3.1.2-1
* Elasticsearch Version: 6.8.3
* MongoDB Version: 1:3.2.11-2+deb9u1
* Operating System: Debian 9.11
* Browser version: not sure if relevant : last firefox version

**Comment by Trapulo** [comment]
same trouble here. A graylog working until yesterday, today report that error if I enter a stream with some data. Emtpy streams do not report any error.

**Comment by Nekomata-Sensei** [comment]
same thing,  happened during regular search activity.
that is fresh GL installation via docker, just few days old.

`e is null
Stack Trace:
_updateHeight@https://log.SOMESERVER.COM/assets/LoggedInPage.af2f821c666e2573f8ad.js:1:187288
componentDidMount@https://log.SOMESERVER.COM/assets/LoggedInPage.af2f821c666e2573f8ad.js:1:186922
_/<@https://log.SOMESERVER.COM/assets/vendor.91c91d4a31d54d96392a.js:73:88989
Ro@https://log.SOMESERVER.COM/assets/vendor.91c91d4a31d54d96392a.js:18:82395
Xo@https://log.SOMESERVER.COM/assets/vendor.91c91d4a31d54d96392a.js:18:85072
Os/<@https://log.SOMESERVER.COM/assets/vendor.91c91d4a31d54d96392a.js:18:98279
29/</exports.unstable_runWithPriority@https://log.SOMESERVER.COM/assets/vendor.91c91d4a31d54d96392a.js:35:3284
Os@https://log.SOMESERVER.COM/assets/vendor.91c91d4a31d54d96392a.js:18:98212
Ys@https://log.SOMESERVER.COM/assets/vendor.91c91d4a31d54d96392a.js:18:97988
Ss@https://log.SOMESERVER.COM/assets/vendor.91c91d4a31d54d96392a.js:18:97335
Ls@https://log.SOMESERVER.COM/assets/vendor.91c91d4a31d54d96392a.js:18:96354
Zo@https://log.SOMESERVER.COM/assets/vendor.91c91d4a31d54d96392a.js:18:95228
enqueueSetState@https://log.SOMESERVER.COM/assets/vendor.91c91d4a31d54d96392a.js:18:44755
24/b.prototype.setState@https://log.SOMESERVER.COM/assets/vendor.91c91d4a31d54d96392a.js:26:1665
_refreshData/this.promise</<@https://log.SOMESERVER.COM/assets/90afab18-75.af2f821c666e2573f8ad.js:1:2875
l@https://log.SOMESERVER.COM/assets/builtins.af2f821c666e2573f8ad.js:104:88608
YPyIcTeH/</</<[22]</n.exports/O.prototype._settlePromiseFromHandler@https://log.SOMESERVER.COM/assets/builtins.af2f821c666e2573f8ad.js:104:61890
YPyIcTeH/</</<[22]</n.exports/O.prototype._settlePromise@https://log.SOMESERVER.COM/assets/builtins.af2f821c666e2573f8ad.js:104:62690
YPyIcTeH/</</<[22]</n.exports/O.prototype._settlePromise0@https://log.SOMESERVER.COM/assets/builtins.af2f821c666e2573f8ad.js:104:63389
YPyIcTeH/</</<[22]</n.exports/O.prototype._settlePromises@https://log.SOMESERVER.COM/assets/builtins.af2f821c666e2573f8ad.js:104:64716
YPyIcTeH/</</<[2]</s.prototype._drainQueue@https://log.SOMESERVER.COM/assets/builtins.af2f821c666e2573f8ad.js:104:18582
YPyIcTeH/</</<[2]</s.prototype._drainQueues@https://log.SOMESERVER.COM/assets/builtins.af2f821c666e2573f8ad.js:104:18643
s/this.drainQueues@https://log.SOMESERVER.COM/assets/builtins.af2f821c666e2573f8ad.js:104:16866`

**Comment by Sanlokii** [comment]
Hi, same problem for me too.

**My environment :**

- Graylog 3.1.3
- Elasticsearch 6.7.2
- MongoDB 4.0.9
- OS Debian 9.9

**Comment by dennisoelkers** [comment]
Thanks for reporting this! It will be fixed in 3.2.0, which is due for release at the end of the month.
